presto-main/src/main/java/com/facebook/presto/split/BufferingSplitSource.java

```diff
@@ -45,6 +45,12 @@ public ConnectorId getConnectorId()
     }
 
     @Override
+    public Split getEmptySplit()
```

Comment:
I tried to find out how to implement it earlier today and it didn't seem to that easy, but it might be because I didn't understand plugins/connectors/catalogs creation code :) Nevertheless my best guess was that it would be more complicated, so if you don't mind, I would prefer to leave it as it is.